SaintCoinach/ex.json

```diff
@@ -1840,7 +2070,7 @@
         },
         {
           "index": 3,
-          "name": "ChocoboRaceAbilityType",
+          "name": "IsActive",
```

Comment:
Wrong name?

Reply:
Yeah changed it as 
IsActive true = useable abilities 
IsActive false = passive abilities

Reply:
That's true for the column *inside* ChocoboRaceAbilityType, but we have to leave the linking name intact here.